server/plugins/DelayUGens.cpp

```diff
@@ -6210,7 +6210,7 @@ void TGrains_next(TGrains *unit, int inNumSamples)
 				if (grain->chan >= (int)numOutputs) grain->chan -= numOutputs;
 			} else {
 				grain->chan = 0;
-				pan = sc_wrap(pan * 0.5f + 0.5f, 0.f, 1.f);
+				pan = sc_clip(pan * 0.5f + 0.5f, 0.f, 1.f);
```

Comment:
2-channel should clip, but more than 2 channel should wrap (line 6204)?

it is both inconsistent and may break existing pieces, relying on the behavior. maybe the documentation should reflect the wrap-around?